packages/mail/USE_CASES.md

```diff
@@ -472,3 +473,119 @@ Find more information about all of SendGrid's whitelabeling related documentatio
 You can find documentation for how to view your email statistics via the UI [here](https://app.sendgrid.com/statistics) and via API [here](https://github.com/sendgrid/sendgrid-nodejs/blob/master/packages/client/USAGE.md#stats).
 
 Alternatively, we can post events to a URL of your choice via our [Event Webhook](https://sendgrid.com/docs/API_Reference/Webhooks/event.html) about events that occur as SendGrid processes your email.
+
+<a name="slackeventintegration"></a>
+## Slack event integration
+- Create your Slack app by going to https://api.slack.com/apps/new
+- Go to "Event Subscriptions" configuration page
+- Turns "Enable Events" option to "on"
+- Now we need an actual endpoint to put in "Request URL" input box for the verification, so you will need to create an app for receiving Slack webhook and make sure your app can be accessed from the Internet
+  - Use any Node.js server of your choice to create an endpoint which accept "POST" method
+  - You can make sure the request is actually from your Slack app by compare the value of "token" key from request body with your "Verification Token" in your app's "App Credentials" section. You could store the token value in your server's environment variable to make it secured.
+
+    example request body
+    ```
```

Comment:
You can add the word 'json' immediately after the backticks here to tell GitHub that this is JSON-syntax code.